pixiedust/display/chart/renderers/baseChartDisplay.py

```diff
@@ -160,7 +160,7 @@ def validateOptions(self):
             del self.options[key]
 
     def getExtraFields(self):
-        return []
+        return self.getNonNumericValueFields()
```

Comment:
@glynnbird baseChartDisplay is used by every single renderer, not just mapbox. Adding non numeric value fields as extra fields all the time will certainly have side effects. Why is that needed here?

Reply:
It's like that because this call is made to get a dataframe prior to rendering:

https://github.com/ibm-watson-data-lab/pixiedust/blob/issue203/pixiedust/display/chart/renderers/mapbox/mapBoxMapDisplay.py#L81

Without the extraFields change, then the non-numeric fields were not present in the Pandas data frame and couldn't be rendered.

Reply:
I see, then the correct usage is to override this method in the Mapbox concrete class.